mlx/backend/cuda/matmul.cpp

```diff
@@ -93,7 +95,25 @@ void gemm_and_bias(
       a_batch_strides.back(),
       b_batch_strides.back());
   if (bias) {
-    gemm.set_bias(encoder, *bias);
+    if (a.dtype() == complex64) {
+      // For complex64, cuBLASLt does not support
+      // bias epilogue; fall back to a separate add kernel.
+      gemm.run(
+          encoder,
+          out,
+          a,
+          b,
+          batch_shape,
+          a_batch_strides,
+          b_batch_strides,
+          alpha);
+
+      array bias_arr = astype(*bias, out.dtype(), s);
+      out = add(out, bias_arr, s);
```

Comment:
This is problematic here. We can't do MLX ops inside a primitive's evaluation.

Instead of doing the fallback this way, just fallback to the full addmm. So the condition could be

```
if (bias && a.dtype() != complex64)
```